packages/core/adder/categories.ts

```diff
@@ -3,9 +3,10 @@ export type CategoryInfo = {
     name: string;
     description: string;
 };
-
+export type CategoryKeys = "styling" | "tools" | "asd";
```

Comment:
this `"asd"` literal seems to have been accidentally left in :p

Reply:
Thanks! I have also properly fixed the error that I had while accidentally adding that!